modules/installation-initializing.adoc

```diff
@@ -254,7 +254,7 @@ ifdef::rhv[]
 For production {product-title} clusters on which you want to perform installation debugging or disaster recovery, select an SSH key that your `ssh-agent` process uses.
 ====
 ... For `Platform`, select `ovirt`.
-... For `Enter oVirt's API endpoint URL`, enter the URL of the {rh-virtualization} API using this format:
+... For `oVirt API endpoint URL`, enter the URL of the {rh-virtualization} API using this format:
```

Comment:
Rolfe, I suggest you download the latest OCP4.6 installer [here](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp-dev-preview/latest-4.6/) and go through the questions one by one. They've changed notably since the early days. If I tried to point out all the differences there, it would be quite a text wall. So I think this approach is more direct. 

Note that you don't actually need to deploy your OCP cluster. Just download the installer, extract it and run `openshift-install create install-config`. You can do that on your workstation without any additional requirements.

Reply:
I'll address this in a separate PR. I have to get this one published asap.

Reply:
Okay. This was actually mi biggest issue in this PR. But if this is being dealt with separately, I'm fine with that.